bert_reranker/data/convert_faq_json.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import json
+import logging
+import random
+
+from tqdm import tqdm
+
+from bert_reranker.data.data_loader import remove_html_toks
+
+logger = logging.getLogger(__name__)
+
+
+def make_qa_pairs_faq(faq_path, n_wrong_answers=2, seed=42):
```

Comment:
is there any way to add an option to output a json format that we can load directly with our main function? (so we can use this as dev set)

Reply:
we could just save the .json in the repo